client/client_test.go

```diff
@@ -210,7 +210,11 @@ func TestUpdateFeatures(t *testing.T) {
 	}`)
 
 	cfg := config.DefaultConfig()
-	cfg.Git.RepoPath = "/tmp"
+	dir, err := ioutil.TempDir("", "example")
```

Comment:
May we generalize these into a shared function, something like this:

```
func TempDir(t *testing.T) string {
	dir, err := ioutil.TempDir(os.TempDir(), "dcdr")
	if err != nil {
		t.Fatalf("ioutil.TempDir error: %s", err.Error())
	}
	return dir
}
```

We can also use `t.Fatalf` to allow the test-suite to handle the error gracefully. Also we should prefix the directory with a related to this project: `dcdr` or similar.

Then from each test case we can call the parent function and not need to handle the errors:
```
dir := TempDir(t)
defer os.RemoveAll(dir)
cfg.Git.RepoPath = dir
```